src/Polly.Core/Simmy/Behavior/BehaviorStrategyOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Polly.Simmy.Behavior;
+
+#pragma warning disable CS8618 // Required members are not initialized in constructor since this is a DTO, default value is null
```

Comment:
Don't suppress this warning? Just make the `Behavior` nullable?

We used the same approach in other built-in strategies. For example:
https://github.com/App-vNext/Polly/blob/5ad71b712844d974c914dc2ce5f7143bc3f7ba6d/src/Polly.Core/Fallback/FallbackStrategyOptions.TResult.cs#L27